### ISSUE TITLE
Beta testing

### ISSUE THREAD

**Issue by ghost** [uBlock-LLC/uBlock#86]
You said that you need beta testers. Count me in.

Is there a special procedure how you're planning to do it?

**Comment by gorhill** [comment]
I have to look into it. I saw that there is an option to publish "draft" releases here on Github.

Another option is the webstore allows to put in an email so only the persons on the list receive the update, I think, I never used it. Ideally, a draft on Github would be best I think, as it doesn't cause a dependency on any one store, and installing such a draft manually in a different folder than the official version may be best, as this won't disturb or mangle the other official installation if something go wrong. This would require manual install, but then, having more technically-minded people installing a draft version for testing is probably a good thing.

**Comment by gorhill** [comment]
New release candidate available at: https://github.com/gorhill/uBlock/releases/tag/0.2.3.2.rc

Whoever wants to test, install manually (so that you keep you current installation undisturbed) in a folder dedicated to testing purpose. Disabled v0.2.3.1, enable v0.2.3.2. See details of release above to see what changed and thus what needs testing.

I will update the assets so the extension is forced to update its assets when it is launched after the first time it is installed. Removing and re-installing the extension will force a fetching of the remote assets, and thus a re-testing of that code.

**Comment by gorhill** [comment]
By the way, I just want all of this to be informal, i.e. anybody can try and test whenever they feel for. Whenever I feel I have a solid release, I will just published an official release.

**Comment by ghost** [comment]
> Removing and re-installing the extension will force a fetching of the remote assets, and thus a re-testing of that code.

I removed and re-installed it twice and the remote assets are reported as up-to-date. At the first time I saw in HTTPSB that the corresponding XHR requests were blocked in chromium-behind-the-scene, so I whitelisted them. After removing and re-installing µBlock a second time the statistics tab in HTTPSB reported that the assets were obviously fetched from original source. So everything seems to work well :-) 

The only thing that surprised me was that I couldn't see in µBlock itself from where the assets were fetched. When I hover the cursor over the various assets in the About tab it always shows your github repository. Shouldn't it show the orignal source?

**Comment by gorhill** [comment]
I fixed a bug in uBlock about not being able to see the latest content when clicking on a list in the _"3rd-part filters"_ tab. I have to admit I was lazy and didn't create an issue for this one. So clicking on an list in that tab is supposed to show you the most recent version, which should be July 20 for _EasyList_. In the about tab I link to Github for convenience, if someone wants to check what was changed -- the format on that page is more technical, so I figured I could link to Github.
